addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/AbstractHomematicGateway.java

```diff
@@ -80,8 +79,7 @@
     private long lastEventTime = System.currentTimeMillis();
     private DelayedExecuter delayedExecutor = new DelayedExecuter();
     private Set<HmDatapointInfo> echoEvents = Collections.synchronizedSet(new HashSet<HmDatapointInfo>());
-    // FIXME: dependency to openHab
-    private final ScheduledExecutorService scheduler = ThreadPoolManager.getScheduledPool("homematicGateway");
+    private ScheduledExecutorService scheduler;
```

Comment:
In https://github.com/openhab/openhab2-addons/pull/1078#issuecomment-229645333 I meant that you should not have a field for the scheduler at all.